cypress/integration/list_spec.js

```diff
@@ -0,0 +1,65 @@
+describe('List user flows', () => {
+
+  beforeEach(() => {
+    cy.loadList()
+  });
+
+  it('The url should be ${url.com}/markets', () => {
+    cy.url().should('include', '/markets')
+  });
+
+  it('It should display the name of the app', 
```

Comment:
We will need an additional test here to check the clickability/return home of the header as well.

Reply:
Would you prefer I add that failing test now or after the feature has been deployed?

Reply:
maybe add it and comment it out until it's built?

Reply:
I committed this